check/configuration/supported/golangci-lint.md

```diff
@@ -0,0 +1,31 @@
+---
+description: golangcI-lint
+---
+
+# golangci-lint
+
+golangci-lint is a linter for infra.
+
+You can enable the golangci-lint plugin with
+
+```shell
+trunk check enable golangci-lint
+```
+
+# Settings
+
+golangci-lint uses the same config files as the 
+upstream [golangci-lint](https://github.com/golangci/golangci-lint) project, so you can continue to use any
+existing configuration files (ex: `.golangci.json`, `.golangci.toml`, `.golangci.yaml`).
+
+Trunk provides a [default configuration](https://github.com/trunk-io/plugins/tree/main/linters/golangci-lint) if your project does not already have one,
+which you can see in our [open source plugins repo](https://github.com/trunk-io/plugins/tree/main).
+Make sure your go version in `go.mod` matches Trunk's go runtime version. At the time of this writing, Trunk's default go runtime version is `1.21.0`. You can find out what it is via `trunk print-config`, and look for the `runtime` section, and you can override the default version in your `trunk.yaml` via:
```

Comment:
Remove the reference to `print-config` here since it's actually just going to be confusing/verbose for users

Make sure your go version in `go.mod` matches Trunk's go runtime version. At the time of this writing, Trunk's default go runtime version is `1.21.0`. You can override the default version in your `trunk.yaml` via: